win/first-run.bat

```diff
@@ -10,13 +10,23 @@ IF NOT EXIST %AppData%\Hush\hush.conf (
     echo addnode=stilgar.leto.net
     echo addnode=zdash.suprnova.cc
```

Comment:
This diff LGTM but we might want to remove this `supernova.cc` default node, as they're the ones hosting the conflicting seed?

Reply:
I don't mind removing it.  AFAIK there is no hard evidence about suprnova being the root cause.